code/game/objects/items/oddities.dm

```diff
@@ -500,3 +500,71 @@
 				oddity_stats[stat] = rand(1, oddity_stats[stat])
 		AddComponent(/datum/component/inspiration, oddity_stats, perk)
 	set_light(2, 1, COLOR_BLUE_LIGHT)
+
+/obj/item/oddity/pendant
+	name = "pendant of Kultainen"
+	desc = "An ornate golden necklace."
+	icon_state = "golden_necklace"
+	slot_flags = SLOT_MASK
+	origin_tech = list(TECH_MATERIAL = 3)
+	matter = list(MATERIAL_GOLD = 4)
+	spawn_frequency = 0
+	spawn_blacklisted = TRUE
+	prob_perk = 0
+	oddity_stats = list(
+		STAT_MEC = 4,
+		STAT_BIO = 4
+	)
+	var/produce_ready = TRUE
+	var/produce_next
+
+/obj/item/oddity/pendant/Destroy()
+	STOP_PROCESSING(SSobj, src)
+	. = ..()
+
+/obj/item/oddity/pendant/Process()
+	if(world.time > produce_next)
+		visible_message(SPAN_NOTICE("[src] stops humming suddenly."))
+		src.desc = "An ornate golden necklace."
+		produce_ready = TRUE
+		STOP_PROCESSING(SSobj, src)
+		. = ..()
+
+/obj/item/oddity/pendant/attack_self(mob/user)
+	if(produce_ready)
+		new /obj/item/golden_leaf(get_turf(src))
+		user.visible_message(SPAN_NOTICE("[user] opens [src], and a little golden leaf falls from it. Pendant closes shut right after."), SPAN_NOTICE("As you open [src], a little golden leaf falls from it. Pendant closes shut right after and start to hum quietly."))
```

Comment:
"[user] opens [src], and a little golden leaf falls from it. **\The [src]** closes shut right after.

Reply:
That was changed from second \the [src] to just "Pendant" because I didn't like the look of double [src] insertion during the tests, but whatever. Replaced.